codecarbon/output.py

```diff
@@ -61,3 +68,25 @@ def out(self, data: EmissionsData):
             if not file_exists:
                 writer.writeheader()
             writer.writerow(data.values)
+
+
+class HTTPOutput(BaseOutput):
+    """
+    Send emissions data to HTTP endpoint
+    """
+
+    def __init__(self, endpoint_url: str):
+        self.endpoint_url: str = endpoint_url
+
+    def out(self, data: EmissionsData):
+        try:
+            payload = dataclasses.asdict(data)
+            payload["user"] = getpass.getuser()
+            resp = requests.post(self.endpoint_url, json=payload, timeout=10)
+            if resp.status_code != 201:
+                LOGGER.warning(
+                    "CODECARBON : HTTP Output returned an unexpected status code: ",
+                    resp,
+                )
+        except Exception as e:
+            LOGGER.error(e)
```

Comment:
Let's log the traceback as well:

```suggestion
            LOGGER.error(e, exc_info=True)
```